### PR TITLE
Add ability to change opacity for compose view

### PR DESCRIPTION
**Pull request by ASalavei**
Usage example: 
```
val viewController = ComposeUIViewController(
    configure = { opaque = false }, // Make compose scene background transparent
    content = ::SomeComposeNode
)
```

Related issue: https://github.com/JetBrains/compose-multiplatform/issues/4108

### CODE REVIEW

File: compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/ComposeUIViewControllerConfiguration.uikit.kt [hunk 1]
```diff
@@ -35,6 +35,11 @@ class ComposeUIViewControllerConfiguration {
 
     @ExperimentalComposeApi
     var platformLayers: Boolean = false
+
+    /**
+     * Determines whether the Compose view should have an opaque background.
+     */
+    var opaque: Boolean = true
```
Comment (MatkovIvan) [inline compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/ComposeUIViewControllerConfiguration.uikit.kt:44]: Please mark it as `@ExperimentalComposeApi` for the first release

Comment (ASalavei) [reply compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/ComposeUIViewControllerConfiguration.uikit.kt:44]: Sure, done!

File: compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeContainer.uikit.kt [hunk 2]
```diff
@@ -273,11 +275,22 @@ internal class ComposeContainer(
...
         ComposeSceneContextImpl(platformContext)
 
-    private fun createSkikoUIView(renderRelegate: RenderingUIView.Delegate): RenderingUIView =
-        RenderingUIView(
-            renderDelegate = renderRelegate,
-            transparency = false,
+    private fun getContentSizeCategory(): UIContentSizeCategory =
+        traitCollection.preferredContentSizeCategory ?: UIContentSizeCategoryUnspecified
+
+    private fun getSystemDensity(): Density {
```
Comment (MatkovIvan) [inline compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeContainer.uikit.kt]: Bad merge (it was removed)

Comment (ASalavei) [reply compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeContainer.uikit.kt]: yep, removed

**Review by MatkovIvan — COMMENTED** [review]
The code LGTM in general, but
1. We need to agree about public API (transparency vs opaque) to keep it harmonized between platforms. cc @igordmn 
2. I'd like to wait a review from @elijah-semyonov since it affects render pipeline/performance

**Comment by ASalavei** [comment]
> 1. We need to agree about public API (transparency vs opaque) to keep it harmonized between platforms. cc @igordmn

I named it `opaque` because it is most commonly used term in iOS (compared to `transparency`)

**Review by elijah-semyonov — APPROVED** [review]
We discussed it in a call, LGTM

**Review by dima-avdeev-jb — APPROVED** [review]

**Review by MatkovIvan — APPROVED** [review]

**Comment by chrisbanes** [comment]
QQ: is there a performance impact from using `opaque = false`? It doesn't look like it from the PR

**Comment by elijah-semyonov** [comment]
@chrisbanes 
Yes, it's marked in the documentation to the added configuration property.
There is an impact, since iOS can't draw perform direct-to-screen drawing when `opaque` is set to false.

**Pull request merged**